daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -193,6 +193,61 @@ APP ID     APP PORT  AGE  CREATED
 nodeapp    3000      16h  2020-07-29 17:16.22
 ```
 
+#### Upgrade from 0.11.x to 1.0.0
```

Comment:
Are these instructions only from 0.11 to 1.0.0 or do they also apply from 0.10 / 0.9? (if so we should change this)
Also, just to make sure users don't think this refers to k8s version, could be good to say "Upgrade from Dapr 0.11.x to 1.0.0"

Reply:
I have tested 0.11 only. technically the upgrade from 0.10, 0.9 to 1.0.0 should work.

Reply:
> could be good to say "Upgrade from Dapr 0.11.x to 1.0.0"
make sense. I updated it.